docs/exp-algo/degreeCentrality.md

```
@@ -0,0 +1,55 @@
+
+This implementation runs inside FalkorDB’s UDF runtime and uses `graph.traverse` to build an in-memory adjacency representation of the supplied node set.
+
+**Important:** Edges discovered via traversal are treated as **undirected** (symmetrized), similar to `exp.louvain` and `exp.leiden`.
+
+## Syntax
+```cypher
+flex.exp.degreeCentrality({nodes: nodes})
+```
+
+## Parameters
+| Parameter | Type | Required | Description |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)